text_mining.py

```diff
@@ -0,0 +1,294 @@
+"""
+    Alex Chapman
+    2/19/17
+
+    Software Design Project 3
+
+    Finding a list of U.S. Representatives and analyzing the sentiment of
+    their respective wikipedia pages in corrolation with party affiliation.
+"""
+
```

Comment:
Is this a duplicate file? Your repository should only contain codes that are part of your final program.